blog/docs/articles/using-sideloaded-images-with-kind.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Using Sideloaded Images with Kind
+linkTitle: Using-Sideloaded -Images-with-Kind
+author: "Ed Jung"
+date: 2022-05-26
+description: How to use sideloaded images with Knative on Kind
+type: "blog"
+---
+
+Once you've setup a kind Knative cluster with the [`quickstart` plugin](https://github.com/knative-sandbox/kn-plugin-quickstart), you may want to try it with images from your developer environment, without an image registry.
+
+This document walks through these steps:
+
+* sideloading an image into kind
+* creating a Knative service that uses the sideloaded image
+
+## Pre-Requisities
+
+This tutorial assumes you have cluster running as from [Getting Started with Knative](https://knative.dev/docs/getting-started/),
+ and will sideload the helloworld-go image, but you may substitute your own image.
+
+
+## Sideload Your Image into the Cluster
+
```

Comment:
```suggestion
## Sideload your image into the cluster

This tutorial sideloads the `helloworld-go` image, but you can substitute your own image.

```

Please always use sentence case for titles, see https://github.com/knative/docs/blob/main/contribute-to-docs/style-guide/style-and-formatting.md#titles-and-headings